### PR TITLE
Jk/editing

### PR DESCRIPTION
**Pull request by JanKoehnlein**

### CODE REVIEW

File: example/states/webview/lerna.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
       "stream": true
     }
-  }
-}
\ No newline at end of file
+  },
+  "packages": [
+    "states-sprotty*"
+  ]
+}
```
Comment (spoenemann) [inline example/states/webview/lerna.json:13]: Could we get rid of Lerna here and move the content of the `states-sprotty-webview` subfolder one level higher?

File: server/codeActions/StatesCodeActionService.xtend [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package io.typefox.examples.theia.states.ide.server.codeActions
```
Comment (spoenemann) [inline server/codeActions/StatesCodeActionService.xtend:1]: Why isn't this Xtend file in `src`?

File: example/states/webview/lerna.json (hunk 1 above)
Comment (JanKoehnlein) [reply example/states/webview/lerna.json:13]: Merged the two webview packages into one and removed lerna

File: server/codeActions/StatesCodeActionService.xtend (hunk 2 above)
Comment (JanKoehnlein) [reply server/codeActions/StatesCodeActionService.xtend:1]: deleted

**Pull request merged**